packages/vscode/src/plugins/prisma-studio/studio-page-draft.ts

```diff
@@ -0,0 +1,229 @@
+        (random() * 100)::numeric(10,4), 
+        random()::real * 10, 
+        random() * 100, 
+        CASE WHEN i % 2 = 0 THEN (random() * 500)::integer ELSE NULL END
+      from generate_series(1, 20) as s(i);
+
+      create table if not exists "public"."json" (
+        "id" serial primary key,
+        "json_col" json,
+        "jsonb_col" jsonb
+      );
+      insert into "public"."json" ("json_col", "jsonb_col")
+      values
+        ('{"hello":"world"}', '{"numbers":[1,2,3]}'::jsonb),
+        ('{"flag":true}', '{"flag":false}'::jsonb);
+
+      create table if not exists "public"."lists" (
+        "id" serial primary key,
+        "integer_list" integer[] not null,
+        "text_list" text[],
+        "boolean_list" boolean[] not null
+      );
+      insert into "public"."lists" ("integer_list", "text_list", "boolean_list")
+      values
+        (ARRAY[1, 2, 3], ARRAY['apple', 'banana', 'cherry'], ARRAY[true, false, true]),
+        (ARRAY[10, 20], NULL, ARRAY[false, false]),
+        (ARRAY[100], ARRAY['single'], ARRAY[true]);
+
+      create type "public"."role" as enum ('admin', 'maintainer', 'member', 'moderator', 'contributor', 'viewer', 'editor', 'guest', 'supervisor', 'developer');
+
+      create table if not exists "public"."logs" (
+        "message" text,
+        "created_at" timestamp default now(),
+        "level" "public"."role", -- Reusing the existing role enum
+        "processed" boolean default false,
+        "details" json,
+        "value" numeric(8, 2)
+      );
+      insert into "public"."logs" ("message", "level", "processed", "details", "value")
+      values
+        ('Initial log entry', 'admin', true, '{"source": "system"}', 123.45),
+        ('User action recorded', 'member', false, '{"user_id": 10, "action": "login"}', 0.00),
+        ('Maintenance task started', 'maintainer', false, null, null);
+
+      create schema if not exists "zoo";
+      create table if not exists "zoo"."animals" ("id" serial primary key, "name" text);
+      insert into "zoo"."animals" DEFAULT VALUES;
+      insert into "zoo"."animals" DEFAULT VALUES;
+      insert into "zoo"."animals" DEFAULT VALUES;
+      
+      create table if not exists "public"."users" ("id" serial primary key, "created_at" timestamp default now(), "activated" boolean default false, "role" role);
+      insert into "public"."users" ("created_at", "activated", "role")
+      select NOW() + (i * INTERVAL '1 millisecond'), (i % 2 = 0), (ARRAY['admin', 'maintainer', 'member', 'moderator', 'contributor', 'viewer', 'editor', 'guest', 'supervisor', 'developer'])[floor(random() * 10 + 1)]::role from generate_series(1, 200) as s(i);
+
+      create table if not exists "public"."posts" ("id" serial primary key, "created_at" timestamp default now(), "user_id" integer not null references "public"."users"("id"), "content" text not null);
+      insert into "public"."posts" ("user_id", "content") values (1, 'Hello world!');
+    \`)
```

Comment:
This huge wall of test is to spin up a pglite db. Will be removed once we support ppg.